tests/integration/test_requests.py

```diff
@@ -114,7 +114,8 @@ def test_post_chunked_binary(tmpdir, httpbin):
     assert req1 == req2
 
 
-@pytest.mark.xfail('sys.version_info >= (3, 6)', strict=True, raises=ConnectionError)
+@pytest.mark.xfail('sys.version_info >= (3, 5)', strict=True, raises=ConnectionError)
+@pytest.mark.xfail((3, 5) > sys.version_info < (3, 6) and platform.python_implementation() == 'CPython', 'Fails on CPython 3.5')
```

Comment:
probably need to import these if platform doesn't work in the pytest.mark strings.